### PR TITLE
Announce Tenzir Node v4.21 and Tenzir Platform v1.1

### PR DESCRIPTION
**Pull request by dominiklohmann**
This collects the blog posts written by the team today.

### CODE REVIEW

File: web/blog/tenzir-node-v4.21/index.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+Tenzir now supports reading Fortinet and Sophos logs. This feature is powered
+by an update to the KV-Parser, which has been upgraded to allow quoted values.
+For example, here's how you can use the new parser to take apart a Fortinet
+FortiGate log file:
+
+```text {0} title="Parse a Fortinet log file"
+from "fortinet.log" read kv
+| parse rawdata kv "\|" "="
+```
```
Comment (mavam) [inline web/blog/tenzir-node-v4.21/index.md:29]: A before and after example is necessary to make the headline credible. One log sample suffices.

**Review by mavam — APPROVED** [review]

File: web/blog/tenzir-platform-v1.1/index.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+- We fixed the incorrect x-axis positioning when chart values are all zeroes.
+- Fixed flashing content after clicking "Load More" in tables and updated the
+  button styling.
+- Resolved a bug preventing pipeline editing in the details view.
+- Improved UX for adding self-hosted nodes—clear instructions are now visible on
+  the pipelines page when a node is created and disconnected.
+
+## Join Us for Office Hours
+
+Every second Tuesday at 8 AM EST / 11 AM EST / 5 PM CET / 9:30 PM IST, we hold
```
Comment (raxyte) [inline web/blog/tenzir-platform-v1.1/index.md:65]: `8:30 PM IST` is the correct time (IST = GMT + 5:30), but can we just use a single zone? @gitryder

File: web/blog/tenzir-node-v4.21/index.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+| …
+```
+
+You can find more information about these new options in [our
+documentation](/formats#parser-schema-inference).
+
+## Azure Blob Storage Integration
+
+Tenzir now has an [integration with Azure Blob
+Storage](/connectors/azure-blob-storage)! This integration enables users to
```
Comment (lava) [inline web/blog/tenzir-node-v4.21/index.md:55]: I think we usually link to `/next` in the blog posts, ie. `/next/connectors/azure-blob-storage`, to make the CI happy and to make the blog post work before the release as well.

**Review by lava — APPROVED** [review]
Looks good to me.

**Pull request merged**